shopify-features/models/profiles.yaml

```diff
@@ -81,7 +81,7 @@ models:
         #days since last seen
         - entity_var:
             name: days_since_last_seen
-            select: datediff(day, date(max_timestamp_bw_tracks_pages),date('{{timestamp.Format("2006-01-02 15:04:05")}}'))
+            select: "{% if !(end_time|isnil) %} datediff(day, date(max_timestamp_bw_tracks_pages),date('{{end_time.Format(\"2006-01-02 15:04:05\")}}') {% else %} 1=1 {% endif %}"
```

Comment:
@shubhammehra4 the open bracket for datediff was not balanced. That's the reason for the error.